README.md

```diff
@@ -99,6 +99,17 @@ class { '::heka':
 }
 ```
 
+####Package versioning
+
+To specify a particular Heka package version to install, use the `package_download_url` and `version` parameters:
```

Comment:
Done in d54b233.

Weirdly enough, specifying `ensure => latest` and pointing to the RPM/DEB file seems ot make Puppet just install it regardless of the version.